state/execution.go

```diff
@@ -348,6 +365,7 @@ func execBlockOnProxyApp(
 	}
 	proxyAppConn.SetGlobalCallback(proxyCb)
 
+	startTime := time.Now()
```

Comment:
This is a little strange. If you want to include `BeginBlock` in the `excTime`, I think it is right to include `EndBlock`. However, I think it is better to measure only `DeliveryTx` purely than that.